src/app/asset-page/licenses.ts

```diff
@@ -7,6 +7,10 @@ export const licenses = [
       link: 'https://creativecommons.org/licenses/by/4.0/',
       img: 'https://licensebuttons.net/l/by/4.0/88x31.png'
     },
+    { name: 'Creative Commons: Free Reuse (CC0)',
```

Comment:
@eslawski reminder to be on the lookout for issues opened related to this license, as some publishers may add the license as something like, just "CCO", as we've seen that happen.

Reply:
👍  Will do. My understanding is that these licenses are now actually a controlled list of options. If these happen to change I would think we would need to talk with the service teams.